#python/exception


---
*wyjątek*/*exception* wydarza się, gdy program napotyka błąd lub nieoczekiwany problem (przykładowe powody: błędne dane wejściowe, błąd syntaktyczny, ...)

Python przerywa działanie programu i wyświetla 










